controllers/utils/utils.go

```diff
@@ -87,17 +85,23 @@ func VerifyIfComponentIsEnabled(ctx context.Context, cli client.Client, componen
 	// there must be only one dsc
 	if len(objectList.Items) == 1 {
 		fields := []string{"spec", "components", componentName, "managementState"}
-		if componentName == KserveAuthorinoComponent {
+		if componentName == KServeWithServiceMeshComponent {
 			// For KServe, Authorino is required when ServiceMesh is enabled
 			// By Disabling ServiceMesh for RawDeployment, it should reflect on disabling
 			// the Authorino integration as well.
```

Comment:
Nit: comment needs to be adjusted

Reply:
To serving,not service mesh :)
I did in the other PR :)